src/lib.rs

```diff
@@ -471,7 +530,7 @@ impl LocDiff {
         &self.file
     }
 
-    /// Calculate the LOC diff.
+    /// Calculates the LOC diff.
```

Comment:
Revert this change.  It does not add anything to the source file but raises an unnecessary grammar issue:  there is no subject, now that you use Indicative instead of Imperative!

Reply:
grammarly does not seem to have a problem with this.

Reply:
In my opinion, this is a phrase without a subject.  You just have a verb and an object.  In Imperative, this would not matter since this mode does not require a subject in each case.

Reply:
Well, I checked it with a grammar checker. So I don't know what to tell you.. 😅